### ISSUE TITLE
az.summary ignores the parameter coords

### ISSUE THREAD

**Issue by impact27** [arviz-devs/arviz#1732]
**Describe the bug**
I have a `arviz.data.inference_data.InferenceData` object with several `'chain'`. I want to select some of them. I use:

`results_summary = az.summary(trace, coords={"chain": [0, 1, 2]})`

But it looks like the `coords` parameter doesn't do anything.

**To Reproduce**
Sample a model with 
```
import pymc3 as pm
[...]
with pm.Model():
    [...]
    trace = pm.sample(return_inferencedata=True, chains=8)
results_summary = az.summary(trace, coords={"chain": [0, 1, 2]})
```

**Expected behavior**
I would expect only the chains 0, 1, 2 to be used for `results_summary `

**Additional context**
Versions of `arviz` and other libraries used, operating system used, and anything else that may be useful.

`az.__version__ = '0.11.2'`
`pm.__version__ = '3.11.2'`

On WIndows 10

**Comment by ahartikainen** [comment]
Until we fix this, for now I think `.sel` methods could be used to select the chains and then you can call summary.

**Comment by OriolAbril** [comment]
It's already fixed in main branch. Was fixed by #1201 iirc
